secure/net/src/network_controller/inbound_handler.rs

```diff
@@ -1,34 +1,29 @@
 // Copyright © Aptos Foundation
 
 use crate::{
-    network_controller::{error::Error, Message, MessageType, NetworkMessage},
-    NetworkServer,
+    grpc_network_service::RemoteExecutionServerWrapper,
+    network_controller::{Message, MessageType},
 };
-use aptos_logger::{error, warn};
+use aptos_logger::warn;
 use crossbeam_channel::Sender;
 use std::{
     collections::HashMap,
     net::SocketAddr,
     sync::{Arc, Mutex},
-    thread,
 };
+use tokio::{runtime::Runtime, sync::oneshot};
 
 pub struct InboundHandler {
     service: String,
-    server: Arc<Mutex<NetworkServer>>,
-    // Used to route incoming messages to correct channel.
+    listen_addr: SocketAddr,
     inbound_handlers: Arc<Mutex<HashMap<MessageType, Sender<Message>>>>,
 }
 
 impl InboundHandler {
-    pub fn new(service: String, listen_addr: SocketAddr, timeout_ms: u64) -> Self {
+    pub fn new(service: String, listen_addr: SocketAddr, _: u64) -> Self {
```

Comment:
remove the timeout if not applicable anymore.. how do we configure the timeout for gRPC btw?

Reply:
Using the timeouts as is. We can set in gRPC when we create the server